docs/deploying-airbyte/on-plural.md

```diff
@@ -25,19 +26,35 @@ To install Airbyte on your Plural repo, simply run:
 plural bundle install airbyte airbyte-aws
 ```
 
+Plural's airbyte distribution currently has support for AWS, GCP and Azure set up and ready to go, feel free to pick whichever best fits your infrastructure.
+
 The CLI will prompt you to choose whether or not you want to use Plural OIDC, which means you're using Plural as your identity provider for SSO.
 
 After this, run:
 
 ```bash
 plural build
-plural deploy --commit "Initial Deploy."
+plural deploy --commit "deploying airbyte"
+```
+
+## Adding the Plural Console
+
+To make management of your installation as simple as possible, including scaling, receiving automated upgrades and getting out-of-the-box dashboarding and log aggregation, we definitely recommending also installing the plural console.  This can be done using the exact same process as above:
```

Comment:
```suggestion
To make management of your installation as simple as possible, we recommend installing the Plural console. The Plural console provides resource scaling, automated upgrades, out-of-the-box dashboarding, and log aggregation. This can be done using the exact same process as above:
```